kit/SlideCompressor.hpp

```diff
@@ -36,7 +36,8 @@ class SlideCompressor {
     {
         auto item = std::make_shared<SlideItem>();
         _items.push_back(item);
-                _pool.pushWork([=]{ workFn(item->_output); });
+        _pool.pushWork([item = std::move(item), workFn = std::move(workFn)]
```

Comment:
I like this, it's easier to reason about the body of the lambda when the capture list is explicit.